bin/peggy-cli.mjs

```diff
@@ -404,7 +412,7 @@ export class PeggyCLI extends Command {
 
     if (this.outputFile === "-") {
       return Promise.resolve(
-        (!this.testFile && !this.testText) ? this.std.out : null
+        (!this.testFile && (typeof this.testText !== "string")) ? this.std.out : null
```

Comment:
The same question about `typeof` check -- it is better to explain it explicitly, why just `!this.testText` is not enough